css/amp.css

```diff
@@ -59,7 +59,7 @@ html.-amp-ios-embed {
 }
 
 /** Wrapper for iOS-Embed-Wrapper mode. */
-i-amp-html-wrapper {
+#i-amp-html-wrapper {
```

Comment:
No, not overridable. `i-amp-` and `-amp-` prefixes are forbidden in selectors of any type.